src/bootstrap.jl

```diff
@@ -15,7 +15,7 @@ Tables.Schema:
  :σ          T
  :β          NamedTuple{β_names}{NTuple{p,T}}
  :se         StaticArrays.SArray{Tuple{p},T,1,p}
- :θ          StaticArrays.SArray{Tuple{p},T,1,k}
+ :θ          StaticArrays.SArray{Tuple{k},T,1,k}
```

Comment:
This was a copy-paste error I introduced, right?

Reply:
Yes.

Sorry for putting that in this PR without explanation.  I was trying to decide if the `StaticArrays` package was still needed and found this while doing a search on the name.

Reply:
I thought as much, no worries!